about.html

```diff
@@ -0,0 +1,121 @@
+      with two orchestras in each city, two conductors (the trombonist Wagner Polistchuk and the timpanist Ricardo 
+      Bologna, who also act as musicians for Osesp), and two soloists (the violinist Soraya Landim and cellist 
+      Jin Joo Doh, who are also members of the Orchestra), each performing in four different cities. The repertoire 
+      of the eight performances will feature works by Beethoven (such as Symphonies No. 2 and No. 4) and Saint-Saëns 
+      (such as his Suite for Cello and Orchestra).</p>
+    <p class="email">Please contact us per Email for any further questions about OSESP activities!</p>
+    <a href="mailto:imprensa@osesp.art.br">OSESP foundation</a>
+  </main>
+  <section class="logo" id="logo">
+    <h2>OSESP new logo</h2>
+    <div></div>
+    <p class="logo_mob">The OSESP brand redesign was awarded a silver star at the 35th Festival of the São Paulo Creative Club</p>
+    <p class="logo_dsk">The OSESP brand redesign was awarded a silver star at the 35th Festival of the São Paulo 
+      Creative Club. The São Paulo State Symphony Orchestra has always been affectionately called 'OSESP' by 
+      the public. The problem is that the previous brand only read 'OSP'. The solution was to simply use the name 
+      by which it was called in the brand, in addition to simplifying its design.</p>
+    <img src="images/osesp.svg" alt="osesp logo">
+  </section>
+  <section class="past" id="past">
+    <h2>See the past OSESP free concerts</h2>
+    <div class="sep"></div>
+    <p>Take a look at the last two OSESP free concerts which took place in Jundiaí and in São Paulo</p>
+    <div class="container">
+      <div class="picture">    
+            <img src="images/osesp_01.jpeg" alt="musicians in the city of Jundiaí"/>
+            <div class="centered_1">OSESP 2018</div>
+            <div class="centered_2">Free concert in Jundiaí</div>
+      </div>
+      <div class="picture">
+          <img src="images/osesp_02.jpeg" alt="São Paulo Room">
+          <div class="centered_1">OSESP 2019</div>
+          <div class="centered_2">Free concert in São Paulo</div>
+      </div>
+    </div>
+  </section>
+  <section class="partner" id="partner">
+    <h2>Partner</h2>
+    <div></div>
+    <ul>
+      <li>
+        <img src="images/mozilla.svg" alt="Mozilla logo">
+      </li>
+      <li>
+        <img src="images/google.svg" alt="Google logo">
+      </li>
+      <li>
+        <img src="images/naver.svg" alt="Naver logo">
+      </li>
+      <li>
+        <img src="images/daum.svg" alt="Daumkakao logo">
+      </li>
+      <li>
+        <img src="images/airbnb.svg" alt="Airbnb logo">
+      </li>
+    </ul>
+  </section>
+  <footer>
+    <img class="logo_mob" src="images/creativecommons.svg" alt="Creative Commons logo">
+    <img class="logo_dsk" src="images/creativecommons_white.svg" alt="Creative Commons logo">
+    <p>2015 Creative Commons Korea. Some Rights Reserved.</p>
+  </footer>
```

Comment:
- [ ] It would be better if you make the footer `width: 100%;` in order to match the design 
![image](https://user-images.githubusercontent.com/61790587/169161984-354902c9-2946-4527-aaf6-ca05d1918cc7.png)